src/utilities/likes-utils.jsx

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+const likeVideo = async (video, setLike, like) => {
+  const token = localStorage.getItem("token");
+  {
+    if (like.find((item) => item._id === video._id)) {
+      alert("this is liked already !!!");
+      return;
+    }
+    try {
+      const { data } = await axios.post(
+        "/api/user/likes",
+        { video },
+        {
+          headers: {
+            authorization: token,
+          },
+        }
+      );
+      setLike(data.likes);
+    } catch (err) {
+      console.log(err);
+    }
+  }
+};
+
+const dislikeVideo = async (id, setLike) => {
+  const token = localStorage.getItem("token");
```

Comment:
you can make a function and use it everywhere

Reply:
Yaa will make a function for this .